src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,41 @@
+.stop-watch {
+  position: relative;
+  display: flex;
+  width: $watch-size;
+  height: $watch-size;
+  border: dotted 5px ;
+  border-radius: 50%;
+
+  &__center {
+    width: $center-size;
+    height: $center-size;
+    background-color: $center-color;
+    border-radius: 50%;
+    margin: auto;
+  }
+
+  &__hand {
+    position: absolute;
+    left: 50%;
+    bottom: 50%;
+    translate: -50%;
```

Comment:
The `translate` property is not standalone. It's a value of `transform` property.

Reply:
Using translate transform shifts the axis of rotation so I use translate alone to rich accuracy.